### PR TITLE
Remove internal links, update network booleans, and clarify UTM availability

### PR DESCRIPTION
**Pull request by dbodow**
**Overview**
- Remove internal document links because APIary does not suppor that currently
- Update Gemini and Google Analytics network booleans for consistency across all platforms
- Clarify that UTMs are only available to Join customers after a successful Join
- Add amazon, pinterest, quora, and spotify network booleans uniformly to metric descriptor endpoints

**Acceptance Criteria**
Paste `apiary.apib` into the [APIary editor](https://app.apiary.io/adstageapi/editor) and verify:
- [ ] Internal links are removed
- [ ] `amazon`, `gemini`, `google_analytics`, `pinterest`, `quora`, and `spotify` Network booleans are documented for all metric descriptor endpoints.
- [ ] UTM availability message is correct
- [ ] No regressions in the rest of the documentation

**Dependent PRs**
Platform https://github.com/AdStage/adstage-platform-v2/pull/4837
Proxy https://github.com/AdStage/adstage-proxy/pull/92

**Review Box**
(Implements changes for network boolean consistency)
http://ec2-107-20-40-91.compute-1.amazonaws.com:3000/

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1980,15 +2054,15 @@ To get a list of conversion columns, see the Build Reports section - "Organizati
...
+    You can use the link to find the appropriate URL for upload in Step 2 of the conversion upload process.
 
     + Attributes
 
         + id: "custom_conversions:14:conversions" (string) - The metric name reports reports will return when providing conversion metrics for this descriptor.
         + `_links` (object)
             + `adstage:custom_conversion_upload` (object)
-                + href: "https://platform.adstage.io/api/organizations/40/metric_descriptors/14/conversions" (string) - The URL for Step 2, which will accept conversion data for this metric descriptor.
+                + href: "https://platform.adstage.io/api/organizations/40/metric_descriptors/14/conversions" (string) - The URL for Step 2, which will accept conversion data for this metric descriptor. Note that because this endpoint is currently equivalent to the V1 endpoint, this link will point to the V1 conversion upload endpoint - if using features from the V2 endpoint, AdStage currently recommends parsing the id number from the `id` response's `id` field, which will allow a seamless upgrade if a V2 endpoint is released, pointing to the V2 upload endpoint.
```
Comment (adstage-jason) [inline apiary.apib]: "... parsing the id number from the `id` response's `id` field, ..." - I think this meant to say "... parsing the id number from the response's `id` field, ..."

File: src/v2/custom_conversions/organization_create_metric_descriptor.apib [hunk 2]
```diff
@@ -35,15 +36,15 @@ To get a list of conversion columns, see the Build Reports section - "Organizati
...
+    You can use the link to find the appropriate URL for upload in Step 2 of the conversion upload process.
 
     + Attributes
 
         + id: "custom_conversions:14:conversions" (string) - The metric name reports reports will return when providing conversion metrics for this descriptor.
         + `_links` (object)
             + `adstage:custom_conversion_upload` (object)
-                + href: "https://platform.adstage.io/api/organizations/40/metric_descriptors/14/conversions" (string) - The URL for Step 2, which will accept conversion data for this metric descriptor.
+                + href: "https://platform.adstage.io/api/organizations/40/metric_descriptors/14/conversions" (string) - The URL for Step 2, which will accept conversion data for this metric descriptor. Note that because this endpoint is currently equivalent to the V1 endpoint, this link will point to the V1 conversion upload endpoint - if using features from the V2 endpoint, AdStage currently recommends parsing the id number from the `id` response's `id` field, which will allow a seamless upgrade if a V2 endpoint is released, pointing to the V2 upload endpoint.
```
Comment (adstage-jason) [inline src/v2/custom_conversions/organization_create_metric_descriptor.apib]: See other comment.

File: apiary.apib [hunk 3]
```diff
@@ -1052,6 +1058,9 @@ Gives information about various URLs associated with an ad.
 
 #### UTM Tracking Information
 
+**Note: UTM tracking information is only available if AdStage Join has been added to your AdStage plan and a Join task has been completed for the entity.
+Currently, Join available only as a paid Add-On.**
```
Comment (adstage-jason) [inline apiary.apib]: Should be "Currently, Join is available..."

File: src/v1/discovering_metrics/meta_fields.apib [hunk 4]
```diff
@@ -181,6 +181,9 @@ Gives information about various URLs associated with an ad.
 
 #### UTM Tracking Information
 
+**Note: UTM tracking information is only available if AdStage Join has been added to your AdStage plan and a Join task has been completed for the entity.
+Currently, Join available only as a paid Add-On.**
```
Comment (adstage-jason) [inline src/v1/discovering_metrics/meta_fields.apib]: "Join is available..."

File: apiary.apib (hunk 3 above)
Comment (dbodow) [reply apiary.apib]: - [ ] typo

File: src/v1/discovering_metrics/meta_fields.apib (hunk 4 above)
Comment (dbodow) [reply src/v1/discovering_metrics/meta_fields.apib]: - [ ] typo

File: src/v2/custom_conversions/organization_create_metric_descriptor.apib (hunk 2 above)
Comment (dbodow) [reply src/v2/custom_conversions/organization_create_metric_descriptor.apib]: - [ ] typo

File: apiary.apib (hunk 1 above)
Comment (dbodow) [reply apiary.apib]: - [ ] typo

Comment (dbodow) [reply apiary.apib]: - [x] fixed

File: src/v2/custom_conversions/organization_create_metric_descriptor.apib (hunk 2 above)
Comment (dbodow) [reply src/v2/custom_conversions/organization_create_metric_descriptor.apib]: - [x] fixed

File: apiary.apib (hunk 3 above)
Comment (dbodow) [reply apiary.apib]: - [x] fix

File: src/v1/discovering_metrics/meta_fields.apib (hunk 4 above)
Comment (dbodow) [reply src/v1/discovering_metrics/meta_fields.apib]: - [x] fix

**Pull request merged**